.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Personally, I think good to keep the dev environment separate from the codebase. Need to do a tidy up, so thinking to add `.vscode` files to `.gitignore` so each user can retain their own editor configs. Thoughts?

Reply:
I think there's a benefit by having some settings shared, to have some baseline consistency for how we run things (still with the flexibility to individually customise locally). Don't feel too strongly about this particular setting - though can see it helping someone later down the line who loads up Unified Codes without much context?

Reply:
Just for context, this is a config file for vscode. Generally editor configs are included in `.gitignore` because it's a bit overkill. IMO the app shouldn't depend on any particular editor, so I think better to put scripts etc. into the `package.json`!

Reply:
Yeah that's fair enough, I can see the argument on that side too - particularly if open sourcing it. I'm also sensing tool/environment consistency/variance is another whole discussion point I probably don't want to get involved in just yet so happy to go with whatever works best for now :)

Reply:
just trying to be helpful. if anyone tries to work on the repo, then having a working debug is useful.
we'll need to discuss another way to share that info. perhaps a section in README for dev-type repos?

Reply:
I agree 100%, didn't want to come across as undermining that contribution. Would definitely agree on updating the `README.md` (or perhaps adding a wiki). @mark-prins @katherine-sussol Which one do you guys think?